src/test/java/com/giovds/AverageAgeMojoTest.java

```diff
@@ -0,0 +1,56 @@
+package com.giovds;
```

Comment:
You might want to add a test to validate it only uses compile time artifacts

Reply:
Absolutely. Added it.